include/SFML/Graphics/Transform.inl

```diff
@@ -104,29 +104,27 @@ constexpr Vector2f Transform::transformPoint(const Vector2f& point) const
 constexpr FloatRect Transform::transformRect(const FloatRect& rectangle) const
 {
     // Transform the 4 corners of the rectangle
-    const std::array points = {transformPoint({rectangle.left, rectangle.top}),
-                               transformPoint({rectangle.left, rectangle.top + rectangle.height}),
-                               transformPoint({rectangle.left + rectangle.width, rectangle.top}),
-                               transformPoint({rectangle.left + rectangle.width, rectangle.top + rectangle.height})};
+    const std::array points = {transformPoint(rectangle.position),
+                               transformPoint(rectangle.position + Vector2f(0.f, rectangle.size.y)),
+                               transformPoint(rectangle.position + Vector2f(rectangle.size.x, 0.f)),
+                               transformPoint(rectangle.position + rectangle.size)};
```

Comment:
If we add some extra accessors, we could do something like:

```suggestion
    const std::array points = {transformPoint(rectangle.getTopLeft()),
                               transformPoint(rectangle.getBottomLeft()),
                               transformPoint(rectangle.getTopRight())),
                               transformPoint(rectangle.getBottomRight())};
```

Maybe something to consider for a future PR.

Reply:
Not an API I'd personally want to see. We define the rectangle in one way and shouldn't need a whole matrix of getter functions, and it would create an asymmetrical API, unless you add setters as well, which I'd be even more against.

Reply:
I think getting the corners of a rectangle is a common enough operation to warrant its own helper function. I've used it quite a lot when building UIs or aligning game elements. I also don't think that "getters" are required to have a symmetric "setter", especially when they're computing a value on the fly.